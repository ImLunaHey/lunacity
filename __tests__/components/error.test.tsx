import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Error from '../../src/components/error';

describe('Error', () => {
    it('it renders an error message', () => {
        const message = 'Not Found';
        const { container } = render(<Error message={message} />);
        expect(container).toMatchSnapshot();
        expect(container).toHaveTextContent(message);
    });
});
