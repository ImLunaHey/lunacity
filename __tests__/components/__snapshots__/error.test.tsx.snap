// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error it renders an error message 1`] = `
<div>
  <div
    class="relative rounded border border-red-500 bg-red-500 px-4 py-3 text-white"
    role="alert"
  >
    <span
      class="block sm:inline"
    >
      Not Found
    </span>
  </div>
</div>
`;
